src/pat/inject/inject.js

```diff
@@ -206,6 +207,14 @@ define([
                     cfg.$target.removeClass('is-dirty');
                 }
 
+                var $form = cfg.$target.parents('form')
+                if ($form.size() !== 0 && cfg.$target.data('initial-value') === undefined) {
+                    cfg.$target.data('initial-value', cfg.$target.html());
+                    $form.on('reset', function() {
+                        cfg.$target.html(cfg.$target.data('initial-value'))
+                    })
+                }
+
```

Comment:
It's not clear to me what's going on here or why this is needed. I'm guessing it's raptor specific? Please add a comment above explaining why this is necessary.